### DISCUSSION TITLE
no response after using "attach to visual studio code"     (question about "dev containers " extension)

### DISCUSSION THREAD

**Discussion by hes666** [microsoft/vscode-discussions#1041] [Extension Show and Tell]
i want use vscode to enter my container, after installing "docker " and "dev containers" extension , i try to use  "attach to visual studio code"  to enter my container , but fail with no response , and i can  use  "attach shell"  to enter successfully.
Anyone know about this question?

**Comment by gjsjohnmurray** [comment]
Please raise this with the extension authors, or use one of the other forums listed in the [READ FIRST](https://github.com/microsoft/vscode-discussions/discussions/5) article pinned at the top of this discussion forum. This forum itself focuses on the needs of developers of extensions.
